tests/consensus_tests/test_strict_mode.py

```diff
@@ -176,7 +176,7 @@ def test_payload_strict_mode_upsert_no_local_shard(tmp_path: pathlib.Path):
 
     set_strict_mode(peer_urls[0], COLLECTION_NAME, {
         "enabled": True,
-        "max_collection_payload_size_bytes": 14000,
+        "max_collection_payload_size_bytes": 15000,
```

Comment:
Is it expected that this also grows along with RocksDB storage itself growing slightly?

Reply:
No particular expectation here.
Just adapting the test to the internals of RocksDB, it is a small increase so I did not think twice about it.